modules/social_features/social_content_block/src/ContentBlockBase.php

```diff
@@ -19,11 +19,21 @@ abstract class ContentBlockBase extends PluginBase implements ContentBlockPlugin
    */
   public function supportedSortOptions() : array {
     return [
-      'created' => 'Last created',
-      'changed' => 'Last updated',
+      'created' => [
+        'label' => 'Last created',
+        'limit' => FALSE,
+      ],
+      'changed' => [
+        'label' => 'Last updated',
+        'limit' => FALSE,
+      ],
       'most_commented' => 'Most commented',
       'most_liked' => 'Most liked',
-      'last_interacted' => 'Last interacted',
+      'last_interacted' => [
+        'label' => 'Last interacted',
+        'limit' => FALSE,
+      ],
+      'trending' => 'Comments and likes',
```

Comment:
@OH17L As we discussed with Jessica could you rename it to the `Most comments and likes` for now?

Reply:
Done.